test/test_cuda.py

```diff
@@ -5728,6 +5728,22 @@ def no_pool():
         self.assertEqual(len(get_cudagraph_segments(pool)), 0)
 
 
+    def test_no_triton_on_import(self):
+        script = """import sys; import torch; torch.rand(2, device='cuda'); exit(2 if "triton' in sys.modules else 0)
```

Comment:
This is obviously syntactically incorrect yet the tests are green...

Reply:
Sure, but I'm confused how the tests are green if one of them supposed to fail?

Reply:
It fails only with the script returning error code 2, not 1. That was fixed to fail with anything nonzero.

Reply:
Yep, but let's simplify it every further: any exception during that `subprocess.check_call` should be treated as test failure.